Dockerfile

```
@@ -29,7 +29,7 @@ RUN apt-get update && apt-get install -y gpg curl wget unzip xz-utils git openss
 
 ## Gradle
 
-RUN apt-get update && apt-get install -y --no-install-recommends openjdk-8-jre-headless gradle && \
+RUN apt-get update && apt-get install -y --no-install-recommends openjdk-11-jre-headless gradle && \
```

```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends gradle && \
```

gradle installs latest stable java version (`default-jre-headless`)